code/modules/research/xenobiology/crossbreeding/destabilized.dm

```diff
@@ -0,0 +1,322 @@
+/*
+Destabilized extracts:
+	Throwable extracts that apply an effect on hit.
+*/
+
+/obj/item/slimecross/destabilized
+	name = "destabilized extract"
+	desc = "It seems ready to burst at any moment."
+	effect = "destabilized"
+	icon_state = "destabilized"
+	amountToCreate = 5
+
+/obj/item/slimecross/destabilized/throw_impact(atom/A, var/datum/thrownthing/T)
+	if(!..()) //For those tactical catches
+		if(hitEffect(A, T.thrower))
+			playsound(get_turf(A), 'sound/misc/splort.ogg', get_volume_by_throwforce_and_or_w_class(), 1, -1)
+			qdel(src)
+
+/obj/item/slimecross/destabilized/proc/hitEffect(atom/A, mob/thrower)
+	return TRUE //But nothing happens...
+
+/obj/item/slimecross/destabilized/grey
+	colour = "grey"
+
+/obj/item/slimecross/destabilized/grey/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	var/mob/living/simple_animal/slime/S = new (get_turf(A))
+	S.rabid = TRUE
+	S.Friends += thrower
+	if(isliving(A))
+		var/mob/living/L = A
+		S.Target = L
+		if(S.CanFeedon(L))
+			S.Feedon(L)
+
+/obj/item/slimecross/destabilized/orange
+	colour = "orange"
+
+/obj/item/slimecross/destabilized/orange/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		if(L.fire_stacks < 1)
+			L.fire_stacks = 1
```

Comment:
A single fire stack deals AWFUL amounts of damage; consider bumping this up to, like, 7 fire stacks (the cap for fire stacks from chaos stands, IIRC) or something.

Reply:
Dark purple already does that, orange is meant to simply ignite you. Those two are sorta designed to work in tandem.